### PR TITLE
SME, QE, and peer review pull request for Bxmsdoc 3592 Designing and building cases for case management for 7.2

### PR DESCRIPTION
**Pull request by michelehaglund**
Cristiano and Lubomir, I have updated the Designing and building cases for case management for 7.2 and also included a new chapter to introduce CMMN support. 

I removed the showcase application chapters and provided links to that new doc where necessary. Note that the links will not work until we publish the docs. Here is the Showcase app doc that you already approved for reference: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3565

Original Jira: https://issues.jboss.org/browse/BXMSDOC-3592
CMMN content Jira: https://issues.jboss.org/browse/BXMSDOC-3097
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-3592/

### CODE REVIEW

File: doc-content/enterprise-only/cases/case-management-stage-activation-and-completion-conditions-proc.adoc [hunk 1]
```diff
@@ -3,24 +3,24 @@
...
 
-.Prerequisite 
+.Prerequisite
 * You have created a case definition in the {CENTRAL} process designer.
 * You have added an ad hoc subprocess to the case definition that is to be used as a stage.
 
 .Procedure
 . With the stage selected on the case design canvas, click image:cases/3898.png[] to open the *Properties* panel on the right.
-. Open the `AdHocActivationCondition` property editor to define an activation condition for the start node. For example, `autostart: true` to make the stage automatically activated when a new case instance is started. 
+. Open the `AdHocActivationCondition` property editor to define an activation condition for the start node. For example, `autostart: true` to make the stage automatically activated when a new case instance is started.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-stage-activation-and-completion-conditions-proc.adoc]: For example, set the AdHoc (or some other verb)

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc [hunk 2]
```diff
@@ -41,12 +31,12 @@ In this example, Aimee is the case `owner`, Katy is the `manager`, and the suppl
...
 +
 image::cases/case-management-showcase-overview.png[]
 
 
 [NOTE]
 ====
-Cases can be started and closed using the Showcase application, but they cannot be reopened using this application. You can only reopen a case using a JMS or REST API call. 
-====
\ No newline at end of file
+Cases can be started and closed using the Showcase application, but they cannot be reopened using this application. You can only reopen a case using a JMS or REST API call.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: Do you need to define JMS?

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc [hunk 3]
```diff
@@ -51,10 +51,10 @@ end
 ----
 +
 This business rule detects when a file named `invalid.pdf` is uploaded to the case file. It then removes the `invalid.pdf` document and creates a new instance of the `Prepare hardware spec` user task.
-. Click *Build*, then *Deploy* to deploy the IT Orders project.
+. Click *Build*, then *Deploy* to deploy the *IT_Orders* project.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: I think I've commented before that I would add another click.
Click Build then click Deploy to ...

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc [hunk 4]
```diff
@@ -51,10 +51,10 @@ end
...
 . Create a file called `invalid.pdf` and save it locally.
 . Create a file called `valid-spec.pdf`and save it locally.
-. Open the IT Orders project in {CENTRAL} under *Menu* -> *Projects* -> *IT Orders*.
+. Open the *IT_Orders* project in {CENTRAL} under *Menu* -> *Projects* -> *IT_Orders*.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: I would rephrase something like:
In {CENTRAL}, go  to *Menu* -> *Projects* -> *IT_Orders*  and open the IT ORDERS project.

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc [hunk 5]
```diff
@@ -75,22 +75,22 @@ image::cases/showcase-start-case.png[]
 +
 image::cases/case-management-invalid-spec.png[]
 +
-Click *Complete*. 
+Click *Complete*.
 +
-Instead of the *Task Inbox* showing the `Prepare hardware spec` as `Complete`, it shows as `Ready`. 
-. In Showcase, click *Refresh* in the upper-right corner. Notice that a `Prepare hardware task` appears in the *Completed* column and another appears in the *In Progress* column. 
+Instead of the *Task Inbox* showing the `Prepare hardware spec` as `Complete`, it shows as `Ready`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: Consider something like:
The value in the  *Task Inbox* for the `Prepare hardware spec` is 'Ready' instead of 'Complete'.

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc [hunk 6]
```diff
@@ -75,22 +75,22 @@ image::cases/showcase-start-case.png[]
...
+. In Showcase, click *Refresh* in the upper-right corner. Notice that a `Prepare hardware task` appears in the *Completed* column and another appears in the *In Progress* column.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: a `Prepare hardware task`message appears...

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc [hunk 7]
```diff
@@ -75,22 +75,22 @@ image::cases/showcase-start-case.png[]
...
 +
 image::cases/case-management-new-spec-task.png[]
 +
 This is because the first `Prepare hardware spec` task has been completed with the specification file `invalid.pdf`. As a result, the business rule causes the task and file to be discarded, and a new user task created.
-. In the {CENTRAL} *Task Inbox*, repeat steps 12 and 13, but upload the `valid-spec.pdf` file instead of `invalid.pdf`. 
-. Log out of {CENTRAL} and log back in again as `Katy`. 
-. Go to *Menu* -> *Track* -> *Task Inbox*. There are two `Manager approval` tasks for Katy, one with the `invalid.pdf` hardware specification file, the other with the `valid-spec.pdf` file. 
+. In the {CENTRAL} *Task Inbox*, repeat steps 12 and 13, but upload the `valid-spec.pdf` file instead of `invalid.pdf`.
+. Log out of {CENTRAL} and log back in again as `Katy`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: ...log back in as Katy. - again is redundant.

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc [hunk 8]
```diff
@@ -75,22 +75,22 @@ image::cases/showcase-start-case.png[]
...
+. Go to *Menu* -> *Track* -> *Task Inbox*. There are two `Manager approval` tasks for Katy, one with the `invalid.pdf` hardware specification file, the other with the `valid-spec.pdf` file.
 . Open, claim, and complete each task:
 .. Check the `approve` box for the task that includes the `valid-spec.pdf` file, then click *Complete*.
 .. Do not check the `approve` box on the task with the `invalid.pdf` file, then click *Complete*.
-. Go to *Menu* -> *Manage* -> *Process Instances* and open the *Order for IT hardware* process instance. 
-. Open the *Diagram* tab. Note that the process both `Order rejected` and `Place order` tasks as completed. 
+. Go to *Menu* -> *Manage* -> *Process Instances* and open the *Order for IT hardware* process instance.
+. Open the *Diagram* tab. Note that the process both `Order rejected` and `Place order` tasks as completed.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: Note that.. I think you are missing some words in this last sentence. I don't get it.

File: assemblies/assembly_designing-and-building-cases/main.adoc [hunk 9]
```diff
@@ -15,22 +15,22 @@ As a developer, you can configure assets in {PRODUCT} for case management using
...
-{PRODUCT} includes the IT Orders sample project in {CENTRAL}. This project is referred to throughout this document to explain case management concepts and provide examples. You can import the project in {CENTRAL}, deploy the project to the {KIE_SERVER}, and start IT Orders cases using this project using the Showcase application.
+{PRODUCT} includes the *IT_Orders* sample project in {CENTRAL}. This project is referred to throughout this document to explain case management concepts and provide examples. You can import the project in {CENTRAL}, deploy the project to the {KIE_SERVER}, and start *IT_Orders* cases using this project using the Showcase application.
 
 Use this document to configure and design case management assets.
 
 
 .Prerequisites
 
-* {EAP_LONG} 7.1.0 installed. See  https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
+* {EAP_LONG} 7.2.0 installed. See  https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
```
Comment (emmurphy1) [inline assemblies/assembly_designing-and-building-cases/main.adoc]: This link gives me a 404

Comment (michelehaglund) [reply assemblies/assembly_designing-and-building-cases/main.adoc]: It will until published

File: doc-content/enterprise-only/cases/case-management-stage-activation-and-completion-conditions-proc.adoc (hunk 1 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-stage-activation-and-completion-conditions-proc.adoc]: Done

File: doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc (hunk 2 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc]: Done

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc (hunk 3 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: Done

File: doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc (hunk 8 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/cases/case-management-using-rules-proc.adoc]: Fixed.
